src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -874,7 +844,14 @@ aida:CompoundJustificationShape
    sh:property aida:RequiredConfidencePropertyShape ;
 
    # may provide an optional source system
-   sh:property aida:SystemPropertyShape .
+   sh:property aida:SystemPropertyShape ;
+
+   # no other properties are allowed
+   sh:closed true ;
+
+   # except shared properties and rdf:type
```

Comment:
Are you resolving more than just AIDA-678 in this PR? Shat is the purpose of this change?

Reply:
This is fixing a bug with SharedJustificationShape. It doesn't have anything to do with AIDA-678.